tests/test_identity_map_client_unit_tests.py

```diff
@@ -1,11 +1,14 @@
+import json
 import unittest
 import datetime as dt
+from unittest.mock import patch, MagicMock
 
 from uid2_client import IdentityMapClient, get_datetime_utc_iso_format
 
 
 class IdentityMapUnitTests(unittest.TestCase):
-    identity_map_client = IdentityMapClient("UID2_BASE_URL", "UID2_API_KEY", "wJ0hP19QU4hmpB64Y3fV2dAed8t/mupw3sjN5jNRFzg=")
+    UID2_SECRET_KEY = "wJ0hP19QU4hmpB64Y3fV2dAed8t/mupw3sjN5jNRFzg="
```

Comment:
Nitpick: Can we change this to

```
base64.b64encode(b"UID2_CLIENT_SECRET").decode()
```

so it's clear it isn't a real secret?

Reply:
That is a good idea. I changed it.